lib/tasks/gather.js

```diff
@@ -1236,9 +1240,13 @@ class TaskGather extends SttTask {
   async _resolve(reason, evt) {
     this.logger.info({evt}, `TaskGather:resolve with reason ${reason}`);
     if (this.needsStt && this.ep && this.ep.connected) {
+      // Fix for https://github.com/jambonz/jambonz-feature-server/issues/1281
+      // We should immediately call stop transcription from gather
+      // so that next gather can start transcription immediately
       this.ep.stopTranscription({
         vendor: this.vendor,
-        bugname: this.bugname
+        bugname: this.bugname,
+        gracefulShutdown: false
```

Comment:
What is the purpose of adding gracefulShutdown to the api call here?  It is not used in Endpoint#stopTranscription

Reply:
It is because:
1/ when JAMBONES_TRANSCRIBE_EP_DESTROY_DELAY_MS is enabled, feature server override stopTranscription function  from endpoint to delay sending stop transcribe command to freeswitch.
2/ In some case: gather verb, pause transcribe, fall over to new recognizer, stopTranscription should not do that kind of delay/gracefullShutdown action.
3/ in overrided stopTranscription , it uses that flag gracefulShuwdown  to do the delay stop command or not